faculty_cli/cli.py

```diff
@@ -1451,3 +1450,147 @@ def dataset_ls(project, prefix, show_hidden):
         prefix, project_id=project_id, show_hidden=show_hidden
     ):
         click.echo(item)
+
+
+@cli.group()
+def template():
+    """Manipulate templates in the Faculty knowledge centre."""
+    pass
+
+
+@template.command()
+def init():
+    """Create a blank template."""
+    print("hello")
+
+
+@template.command()
+@click.argument(
+    "source_template_name"
+)  # Outstanding question as to whether this should also alllow the template_ID
+@click.option(
+    "--version",
+    default=None,
+    help="The specific version of the template to use.",
+)
+@click.option(
+    "--directory",
+    default=Path.cwd(),
+    help="The optional target directory where the template will be placed.",
+    type=click.Path(),
+)
+def clone(source_template_name, version, directory):
+    """Clone an existing template."""
+    print(source_template_name)
+    print(directory)
+
+    if version is None:
+        # TODO: Get the latest version
+        print(version)
+
+
+@template.command()
+@click.argument("project_identifier")
+@click.option(
+    "--source_directory",
+    default=Path.cwd(),
+    type=click.Path(),
+    help="Path to the source template directory where the template"
+    + "will be placed.",
+)
+@click.option(
+    "--target-directory",
+    default=Path("/"),
```

Comment:
We should be careful about what this default target directory should be. In various places in the platform or the CLI, we ask users to treat either "/" or "/project" as the root of the workspace. I would like us to have a conversation with Tomas at some point so we make the choice of how this CLI / Kanto treats this argument carefully.